agcaballero/portfolio/src/main/webapp/script.js

```diff
@@ -72,6 +72,129 @@ function testRandomPost(trials = 10000) {
 
 testRandomPost(10000);  // runs 10000 trials on randomPost()
 
+/** Creates a Map using Map API */
+function createMap() {
+  const europeLatLng =
+      new google.maps.LatLng({lat: 48.499998, lng: 23.3833318});
+  // the locations of the photos
+  const places = [
+    {
+      title: 'The Alps',
+      lat: 46.8876,
+      lng: 9.6570,
+      img: 'alps.jpg',
+    },
+    {
+      title: 'Annecy',
+      lat: 45.8992,
+      lng: 6.1294,
+      img: 'annecy.jpg',
+    },
+    {
+      title: 'Chamonix',
+      lat: 45.9237,
+      lng: 6.8694,
+      img: 'chamonix.jpg',
+    },
+    {
+      title: 'Ponte Vecchio, Florence',
+      lat: 43.7679,
+      lng: 11.2531,
+      img: 'florence.jpg',
+    },
+    {
+      title: 'Cliffs of Moher',
+      lat: 52.9715,
+      lng: -9.4309,
+      img: 'moher.jpg',
+    },
+    {
+      title: 'Haleakala Crater, Maui',
+      lat: 20.7097,
+      lng: -156.2535,
+      img: 'haleakala.jpg',
+    },
+    {
+      title: 'James Joyce Bridge, Dublin',
+      lat: 53.3466,
+      lng: -6.2825,
+      img: 'dublin.jpg',
+    },
+    {
+      title: 'Hanauma Bay, Oahu',
+      lat: 21.2690,
+      lng: -157.6938,
+      img: 'hanauma-bay.jpg',
+    },
+    {
+      title: 'Kreuzberg Monastery',
+      lat: 50.370833,
+      lng: 9.968333,
+      img: 'kreuzberg.jpg',
+    },
+    {
+      title: 'Westminster Bridge, London',
+      lat: 51.5009,
+      lng: 0.1220,
+      img: 'london.jpg',
+    },
+    {
+      title: 'Radcliffe Camera, Oxford',
+      lat: 51.7534,
+      lng: -1.2540,
+      img: 'oxford.jpg',
+    },
+    {
+      title: `St. Peter's Cathedral, Rome`,
+      lat: 41.9022,
+      lng: 12.4539,
+      img: 'rome.jpg',
+    },
+    {
+      title: 'Tj&ouml;rn',
+      lat: 58.009423,
+      lng: 11.651235,
+      img: 'tjorn.jpg',
+    },
+    {
+      title: 'Venice',
+      lat: 45.4408,
+      lng: 12.3155,
+      img: 'venice.jpg',
+    },
+  ];
+
+  const map = new google.maps.Map(
+      document.getElementById('map'), {center: europeLatLng, zoom: 4});
+
+  for (let i = 0; i < places.length; i++) {
+    const place = places[i];
+    createMarkerForDisplay(map, place.title, place.img, place.lat, place.lng);
+  }
+}
+
+/** Creates a marker that shows a read-only info window when clicked. */
```

Comment:
What kind of info is displayed?

Reply:
It displays the title of the place and the image taken there. I will update the description to include that.

Reply:
Done